example/adapters/sequelize.js

```diff
@@ -0,0 +1,105 @@
+/*
+ * This is a very rough-edged example, the idea is to still work with the fact that oidc-provider
+ * has a rather "dynamic" schema. This example uses sequelize with sqlite, and all dynamic data
+ * uses JSON fields. id is set to be the primary key, grantId should be additionaly indexed for
+ * models where these fields are set.
+*/
+
+const Sequelize = require('sequelize'); // eslint-disable-line import/no-unresolved
+
+const sequelize = new Sequelize('databaseName', 'username', 'password', {
+  host: 'databaseHost',
+  dialect: 'postgres',
+});
+
+const models = [
+  'Session',
+  'AccessToken',
+  'AuthorizationCode',
+  'RefreshToken',
+  'DeviceCode',
+  'ClientCredentials',
+  'Client',
+  'InitialAccessToken',
+  'RegistrationAccessToken',
```

Comment:
missing `Interaction`, `ReplayDetection`, and `PushedAuthorizationRequest`, see adapter [scaffold](https://github.com/panva/node-oidc-provider/blob/v6.16.0/example/my_adapter.js).

Reply:
I miss them, since in my other project Sequelize definition models are exported from other directory. I added already.